src/libraries/Microsoft.Extensions.Hosting/src/Internal/HostingEnvironment.cs

```diff
@@ -1,15 +1,17 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics;
 using Microsoft.Extensions.FileProviders;
 
 namespace Microsoft.Extensions.Hosting.Internal
 {
-#pragma warning disable CS0618 // Type or member is obsolete
     /// <summary>
     /// This API supports infrastructure and is not intended to be used
     /// directly from your code. This API may change or be removed in future releases.
     /// </summary>
+    [DebuggerDisplay("ApplicationName = {ApplicationName}, EnvironmentName = {EnvironmentName}")]
```

Comment:
```suggestion
    [DebuggerDisplay("ApplicationName = {ApplicationName}, EnvironmentName = {EnvironmentName}, ContentRootPath = {ContentRootPath}")]
```

Reply:
I didn't add the path because I think the result would be too long and cut off. Showing a cut-off path seems like it would be actively unhelpful.